src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -4603,6 +4606,15 @@ public static final void parseAvailableCombatSkills(String responseText) {
           Preferences.setInteger("gladiatorBallMovesKnown", skillId - 7090);
         }
       }
+      if (skillId == SkillPool.BACK_UP) {
+        Matcher matcher = FightRequest.BACKUP_CAMERA_USES.matcher(m.group(2));
+
+        if (matcher.find()) {
+          int limit = KoLCharacter.inRobocore() ? 16 : 11;
+
+          Preferences.setInteger("_backUpUses", limit - StringUtilities.parseInt(matcher.group(1)));
```

Comment:
Could you not achieve this with:

```
COMBAT_SKILL	Back-Up to your Last Enemy	Back-Up to your Last Enemy \((\d+) uses? today\)	_backUpUses=[(11+path(You Robot)*5)-$1]
```

If not, we just need to make sure `path` is available to that sort of Expression

Reply:
Thanks for the suggestion!